index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navItems">
+     
+        
+        <div class="images">
+         <div class="space">
+            <div class="image01"><img src="assets/Shape.png" class="shape00"></div>
+            <div class="image02"><img src="assets/Shape (1).png" class="shape01"></div>
+            </div>
+            <div class="menuitem">
+             <div class="boxic"><img src="assets/ic_menu.svg" class="ic_menu"></div>  
+               </div>
```

Comment:
Kindly add the image as a background image in your CSS file instead. You can use the background-image class in CSS to do this. The hamburger button should be on the right of the header and it should have a `nav` tag.